src/arch/Sound/RageSoundDriver_Generic_Software.cpp

```diff
@@ -55,9 +55,10 @@ RageSoundMixBuffer &RageSoundDriver::MixIntoBuffer( int iFrames, std::int64_t iF
 {
 	ASSERT_M( m_DecodeThread.IsCreated(), "RageSoundDriver::StartDecodeThread() was never called" );
 
-	if( iFrameNumber - iCurrentFrame + iFrames > 0 )
+	const int frameDifference = iFrameNumber - iCurrentFrame + iFrames;
```

Comment:
While not necessarily bad, I think we can just drop the const qualifier. It's just a temp variable anyways to encapsulate the arithmetic.